cmd/virt-handler/virt-handler.go

```diff
@@ -236,6 +357,42 @@ func (app *virtHandlerApp) AddFlags() {
 		"Number of devices to register with Kubernetes device plugin framework")
 }
 
+func (app *virtHandlerApp) setupTLS() error {
+
+	clientCert, err := tls.X509KeyPair(app.clientCertBytes, app.clientKeyBytes)
+	if err != nil {
+		return err
+	}
+	serverCert, err := tls.X509KeyPair(app.serverCertBytes, app.serverKeyBytes)
+	if err != nil {
+		return err
+	}
+
+	caCert, err := cert.ParseCertsPEM(app.serverCertBytes)
+	if err != nil {
+		return err
+	}
+
+	certPool := x509.NewCertPool()
+
+	for _, crt := range caCert {
+		certPool.AddCert(crt)
+	}
+
+	app.migrationTLSConfig = &tls.Config{
+		RootCAs: certPool,
```

Comment:
RootCAs are used to validate server connections. 

I'm pretty sure we need to set ClientCAs as well in order to verify client connections when "RequireAndVerifyClientCert" is used.

Reply:
Done.